Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/ManipulatorView.cpp

```diff
@@ -366,7 +366,10 @@ namespace AzToolsFramework
         cameraCorrectAxisPartialFn(axis1 * axis1.Dot(m_offset), m_cameraCorrectedOffsetAxis1);
         cameraCorrectAxisPartialFn(axis2 * axis2.Dot(m_offset), m_cameraCorrectedOffsetAxis2);
 
-        const auto cameraCorrectedVisualOffset = (m_cameraCorrectedOffsetAxis1 + m_cameraCorrectedOffsetAxis2);
+        const AZ::Vector3 totalScale =
+            manipulatorState.m_nonUniformScale + AZ::Vector3(manipulatorState.m_worldFromLocal.GetUniformScale());
```

Comment:
Is it correct to add the scales rather than multiply them?

Reply:
Argh you're totally right, my bad, will fix 👍

Reply:
I'm going to look at adding some tests for this too as it should be possible to use a mock/stub of the DebugDisplayRequestBus (or just manipulator bounds) to check the view is drawing in the right place based on the scale etc..